scss/_teamshares.scss

```diff
@@ -1104,4 +1106,98 @@ clipboard-copy {
   &.space-between {
     justify-content: space-between;
   }
+
+}
+
+a.button-view-component,
+a.icon-button-view-component {
+  display: block;
+  width: max-content;
+}
+
+.button-view-component {
+  @apply rounded-full border;
+
+  &.small {
+    @apply px-3 py-1 ts-body-3;
+  }
+
+  &.medium {
+    @apply px-6 py-2 ts-body-2;
+  }
+
+  &.large {
+    @apply px-8 py-3 ts-body-2;
+  }
+
+  &.default {
+    @apply bg-gray-100 font-semibold ts-text-default border-gray-300 hover:bg-gray-200 active:bg-gray-300;
+  }
+
+  &.outlined {
+    @apply bg-transparent font-semibold ts-text-default border-gray-400 hover:bg-gray-200 active:bg-gray-300;
+  }
+
+  &.primary {
+    @apply bg-blue-600 font-semibold ts-text-light border-blue-600 hover:bg-blue-700 active:bg-blue-800;
+  }
+
+  &.warning {
+    @apply bg-yellow-500 font-semibold ts-text-default border-yellow-500 hover:bg-yellow-600;
+
+    &:active {
+      filter: brightness(95%);
+    }
+  }
+
+  &.destructive {
+    @apply bg-red-700 font-semibold ts-text-light border-red-700 hover:bg-red-800 active:bg-red-900;
+  }
```

Comment:
Super minor, but seems like they all have `font-semibold` -- can we extract that to the top-level `.button-view-component`?